rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -26,7 +34,20 @@
       "title": "Name",
       "description": "",
       "type": "string",
-      "minLength": 1
+      "minLength": 1,
+      "form": {
+        "focus": true,
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {
+              "term": "name"
```

Comment:
useless but are you sure that the name property is defined as keywork in the ES mapping? As I saw you need to create a new es property with a copy_to. See https://github.com/rero/rero-ils/blob/0d06cbfaef8343d505b2261d44242da8f0509570/rero_ils/modules/patron_types/mappings/v6/patron_types/patron_type-v0.0.1.json#L20 for more details.

Reply:
I think that will be useful to avoid duplicate vendors. I created a new property in ES mapping.